test/utils.py

```diff
@@ -444,6 +444,9 @@ def sample_format(self) -> str:
     },
 )
 
+# Note that the file itself is s32 sample format, but the reference frames are
+# stored as fltp. We can add the s32 original reference frames once we support
+# decoding to non-fltp format, but for now we don't need to.
```

Comment:
I haven't yet explained how the audio references were created. That'll come soon, just know that I have checked for exact equality with the output of TorchAudio before checking anything in.